cfme/fixtures/v2v.py

```diff
@@ -41,6 +57,183 @@ def conversion_tags(appliance, host_creds):
+        'datastore': {
+            'Cluster ({})'.format(provider.data.get('clusters')[0]): {
+                'mappings': [_form_data_mapping('datastores', second_provider, provider,
+                                                request.param[0][0], request.param[0][1]),
+                             _form_data_mapping('datastores', second_provider, provider,
+                                                request.param[1][0], request.param[1][1])]
+            }
+        },
+        'network': {
+            'Cluster ({})'.format(provider.data.get('clusters')[0]): {
+                'mappings': [_form_data_mapping('vlans', second_provider, provider,
+                                                second_provider.data.get('vlans')[0],
+                                                provider.data.get('vlans')[0])]
+            }
+        }
+    })
+    return form_data
+
+
+@pytest.fixture(scope='function')
+def form_data_vm_obj_single_datastore(request, appliance, second_provider, provider):
+    """Return Infra Mapping form data and vm object"""
+    form_data = _form_data(second_provider, provider)
+    recursive_update(form_data, {
+        'general': {
+            'description': "Single Datastore migration of VM from {ds_type1} to {ds_type2},"
+                     .format(ds_type1=request.param[0], ds_type2=request.param[1])},
+        'datastore': {
+            'Cluster ({})'.format(provider.data.get('clusters')[0]): {
+                'mappings': [_form_data_mapping('datastores', second_provider, provider,
+                                                request.param[0], request.param[1])]
+            }
+        }
+    })
+    vm_obj = get_vm(request, appliance, second_provider, request.param[2], request.param[0])
+    return form_data, [vm_obj]
+
+
+@pytest.fixture(scope='function')
+def form_data_vm_obj_single_network(request, appliance, second_provider, provider):
+    form_data = _form_data(second_provider, provider)
+    recursive_update(form_data, {
+        'general': {
+            'description': "Single Network migration of VM from {vlan1} to {vlan2},".
+                     format(vlan1=request.param[0], vlan2=request.param[1])},
+        'network': {
+            'Cluster ({})'.format(provider.data.get('clusters')[0]): {
+                'mappings': [_form_data_mapping('vlans', second_provider, provider,
+                                                request.param[0], request.param[1])]
+            }
+        }
+    })
+    vm_obj = get_vm(request, appliance, second_provider, request.param[2])
+    return form_data, [vm_obj]
+
+
+def _form_data_mapping(selector, second_provider, provider, source_list=None, target_list=None):
+    if selector is not 'clusters':
+        source_data = second_provider.data.get(selector, [])
+        target_data = provider.data.get(selector, [])
+
+        if selector is 'datastores':
+            source = [d.name for d in source_data if d.type == source_list][0]
```

Comment:
Looks like the reference to the 0th element here is misplaced, as there's another reference to it on L226 after confirming it exists.